Sources/Sentry/Public/SentrySpanContext.h

```diff
@@ -78,6 +88,19 @@ SENTRY_NO_INIT
  */
 - (instancetype)initWithOperation:(NSString *)operation sampled:(SentrySampleDecision)sampled;
 
+/**
```

Comment:
Same as in https://github.com/getsentry/sentry-cocoa/pull/4751#discussion_r1930846919